packages/plugin/src/Bundles/Fields/Validation/FileUploadValidation.php

```diff
@@ -58,7 +57,7 @@ public function __construct(
     public function uploadFiles(ValidationEvent $event): void
     {
         $form = $event->getForm();
-        $uploadFields = $form->getLayout()->getFields(FileUploadInterface::class);
+        $uploadFields = $form->getLayout()->getFields(FileUploadField::class);
```

Comment:
@gustavs-gutmanis 

See this comment: https://github.com/solspace/craft-freeform/pull/804#issuecomment-1699694630

I changed to `FileUploadField` in an attempt to exclude `FileDragAndDropField` but it doesn't make any difference as `FileDragAndDropField` extends `FileUploadField`.

So it can be changed back to `FileUploadInterface`.